### PR TITLE
Updated the README and removed the setup script.

### PR DESCRIPTION
**Pull request by rimey**

### CODE REVIEW

**Comment by rimey** [comment]
Note that these updated instructions presume #115.

File: README.md [hunk 1]
```diff
@@ -1,120 +1,67 @@
...
-
-The provided installation script will configure, install, and run cluster-insight. You should be able to run it on any Linux or MacOSX machine. Note, however, that it has only been tested on Ubuntu 14.04 and MacOSX Yosemite.
-
-### Kubernetes API version
-
-The cluster-insight service currently supports only Kubernetes API version "v1".
-To check which API versions are supported by your cluster, run the following command on the master node of your cluster:
-```
-$ kubectl api-versions
 ```
```
Comment (supriyagarg) [inline README.md]: Adding a comment that shows the user how to have the correct context for the project/cluster would be useful. For e.g.

`$ kubectl config use-context gke_<project-id>_<zone-name>_<cluster-name>`

File: README.md [hunk 2]
```diff
@@ -1,120 +1,67 @@
...
 ## REST API
 
-These are the APIs that clients can use to get context graphs and raw resource metadata:
+Cluster Insight makes available the following endpoints:
 
+* `/` returns a help page with links to the following.
 * `/cluster` returns a context graph. The format of the context graph is described below.
-* `/cluster/resources/TYPE` returns the raw metadata for all cluster resources of type TYPE, where TYPE is one of { Node, Pod, Service, ReplicationController, Container, Image, Process }.
+* `/cluster/resources` returns all of the resources (nodes), but not the relations (edges).
+* `/cluster/resources/TYPE` returns the raw metadata for all cluster resources of type TYPE, where TYPE is `nodes`, `pods`, `services`, or `rcontrollers`.
```
Comment (supriyagarg) [inline README.md]: More a comment on collector.py - but why not support /cluster/resources/TYPE calls for  containers, and images? They already exist in /cluster.

**Comment by supriyagarg** [comment]
@rimey: Overall, the readme looks good. Just a couple of comments. It makes more sense to address the API related comment (for containers/images) in a separate PR.

File: README.md (hunk 2 above)
Comment (rimey) [inline README.md]: The /cluster/resources/TYPE calls exist (I have been told) in order to enable manual testing of the raw metadata collection upon which the graph construction is based. For example, the collector.get_pods() handler exists only to make it possible to invoke kubernetes.get_pods() independently of the graph construction code. Containers (or images) are different, in that there is no similar kubernetes.get_containers(). Supporting /cluster/resources/containers would require adding code for that purpose, which doesn't seem worthwhile.

To put it another way, the right thing to do would be to remove the support for /cluster/resources/TYPE altogether, but there's no hurry. The same information is available from /get/TYPE on the Kubernetes API.

Comment (supriyagarg) [inline README.md]: Okay, removing the /TYPE API calls is also fine.

**Comment by rimey** [comment]
I added a Preliminaries section with some tips on configuring `kubectl` (e.g., how to use `kubectl config use-context`).
